src/models/OrgSettings.ts

```diff
@@ -0,0 +1,23 @@
+export default interface OrgSettings {
+  status: ServiceStatus;
+  emergenciesEnabled: boolean;
+  messageOfTheDay: MessageOfTheDay;
+}
+
+export interface MessageOfTheDay {
+  message: string;
+  dateRange: DateRange;
```

Comment:
nullable

Reply:
In the date range are startDate and endDate also nullable? Or just dateRange generaly?

Reply:
just `dateRange` - we decided that if a start date is provided an end date must also be provided and vice versa right?

Reply:
Ok perfect thats the current implementation